client/my-sites/stats/mini-carousel/index.jsx

```diff
@@ -57,6 +73,23 @@ const MiniCarousel = ( { slug, isOdysseyStats } ) => {
 
 	const blocks = [];
 
+	if ( showWriteAPostBanner ) {
+		blocks.push(
+			<MiniCarouselBlock
+				clickEvent={ EVENT_NO_CONTENT_BANNER_CLICK }
+				dismissEvent={ EVENT_NO_CONTENT_BANNER_DISMISS }
+				image={ <img src={ writePost } alt="" width={ 45 } height={ 45 } /> }
+				headerText={ translate( 'Start writing your first post to get more views' ) }
+				contentText={ translate(
+					'Sites with content get more visitors. We’ve found that adding some personality and introducing yourself is a great start to click with your audience.'
+				) }
+				ctaText={ translate( 'Write a post' ) }
+				href={ `/post/${ slug }` }
+				key="write-a-post"
```

Comment:
I see a console.log error from the Blaze banner, but [you already fixed it here](https://github.com/Automattic/wp-calypso/pull/72336/files#diff-5e054664a8d1a6289f819db5d33ffe72c5a9c3e1de7dbf0a981e11c463409316R71)